client/ratings_reviews/components/ReviewFormModal.jsx

```diff
@@ -155,20 +180,21 @@ class ReviewFormModal extends React.Component {
           <form onSubmit={this.handleSubmit}>
             <div className='review-form-modal-body'>
               <div>
-                <label class='review-form-sub-heading'>Overall rating:</label><span id='review-form-rating' class='review-form-invalid-warning'></span>
-                <div>
-                  <select id='review-form-overall-rating' name='rating' value={this.state.rating} onChange={this.handleChange}>
-                    <option value=''>--Please choose an option--</option>
-                    <option value={5}>5</option>
-                    <option value={4}>4</option>
-                    <option value={3}>3</option>
-                    <option value={2}>2</option>
-                    <option value={1}>1</option>
-                  </select>
+                <label class='review-form-sub-heading'>Overall rating*</label><span id='review-form-rating' class='review-form-invalid-warning'></span>
+                <div id='review-form-star-container'>
+                  <div class='review-form-star-rating'>
+                    <input id='star5' name='star' type='radio' value='5' class='radio-btn review-form-star-hide' onClick={this.handleStar}/><label for='star5' >☆</label>
```

Comment:
Also wonder if these star inputs could be created using a for loop or something of that kind

Reply:
Great point! I will some some refactoring later :)